### DISCUSSION TITLE
BUG REPORT - about.html overwritten (1.2.0alpha)

### DISCUSSION THREAD

**Discussion by hollisticated-horse** [donuts-are-good/bearclaw#23] [Bearclaw Discussions]
**What happened?**
When running bearclaw 1.2.0 alpha after setup, the about.html page is overwritten and is empty.
**What should have happened?**
If already present, should either not be changed or changed according to a set md.file that corresponds to the about.html page content.
**More information:**

**Comment by donuts-are-good** [comment]
> If already present, should either not be changed or changed according to a set md.file that corresponds to the about.html page content.

The about.html being re-generated each run is the intended behavior, as that is where the attribution for plugins in-use goes, and adding a new plugin 30 posts into the life of the blog should add that attribution as well. Without adding extra complexity, that was the best means I could think of at the time when implementing plugins. I am open to other ideas.

Perhaps, to achieve what you're describing, I can solve this issue by creating a "now" plugin for you which would give you a preset file you can fill out that then slips itself in at build time to make your `about.html`. Would that be an acceptable compromise?

**Comment by hollisticated-horse** [comment]
I'm confused as to how plugins come into play for the about.html ?

**Comment by donuts-are-good** [comment]
about.html currently is implemented to include attribution for the developers that created the plugins your blog uses (if it uses any) below the content of the about.html page. If no plugins are used, it should display no plugins in the list- I can double check this behavior, but there shouldn't be a plugin credits section if there are no plugins in use. 

I won't always be the author of the plugins someone uses, and though I might not lose sleep over someone pulling my license off a plugin, I can't speak for other developers or the licenses they use on their software. Being able to reproduce the license text, sometimes, is a stipulation of those licenses, and the likeliness of a plugin dev being newer in their career means it probably could stand to help them get some small degree of recognition, which helps you get better plugins through more enthusiastic plugin maintainers.

Maybe it's just the coffee causing me to overthink it, but it felt like the right thing to do.

**Comment by donuts-are-good** [comment]
^ this response doesn't mean this is a wontfix, it's just to convey context of the intent for the current build to answer the question

**Comment by donuts-are-good** [comment]
moving to discussions to give it room to develop
